unstructured/partition/ocr.py

```diff
@@ -385,84 +430,97 @@ def zoom_image(image: PILImage, zoom: float = 1) -> PILImage:
+        ocr_text = unstructured_pytesseract.image_to_string(
             np.array(image),
             lang=ocr_languages,
-            output_type=Output.DATAFRAME,
         )
-        ocr_df = ocr_df.dropna()
-
-        # tesseract performance degrades when the text height is out of the preferred zone so we
-        # zoom the image (in or out depending on estimated text height) for optimum OCR results
-        # but this needs to be evaluated based on actual use case as the optimum scaling also
-        # depend on type of characters (font, language, etc); be careful about this
-        # functionality
-        text_height = ocr_df[TESSERACT_TEXT_HEIGHT].quantile(
-            env_config.TESSERACT_TEXT_HEIGHT_QUANTILE,
-        )
-        if (
-            text_height < env_config.TESSERACT_MIN_TEXT_HEIGHT
-            or text_height > env_config.TESSERACT_MAX_TEXT_HEIGHT
-        ):
-            # rounding avoids unnecessary precision and potential numerical issues assocaited
-            # with numbers very close to 1 inside cv2 image processing
-            zoom = np.round(env_config.TESSERACT_OPTIMUM_TEXT_HEIGHT / text_height, 1)
-            ocr_df = unstructured_pytesseract.image_to_data(
-                np.array(zoom_image(image, zoom)),
-                lang=ocr_languages,
-                output_type=Output.DATAFRAME,
-            )
-            ocr_df = ocr_df.dropna()
+    return ocr_text
 
-        ocr_layout = parse_ocr_data_tesseract(ocr_df, zoom=zoom)
-    return ocr_layout
 
-
-def get_ocr_text_from_image(
+def get_ocr_layout_from_image(
     image: PILImage,
     ocr_languages: str = "eng",
     ocr_agent: str = "tesseract",
-) -> str:
+) -> List[TextRegion]:
     """
-    Get the OCR text from image as a string with paddle or tesseract.
+    Get the OCR regions from image as a list of text regions with paddle or tesseract.
     """
-    if ocr_agent == "paddle":
-        logger.info("Processing entrie page OCR with paddle...")
-        from unstructured.partition.utils.ocr_models import paddle_ocr
-
-        # TODO(yuming): pass in language parameter once we
-        # have the mapping for paddle lang code
-        ocr_data = paddle_ocr.load_agent().ocr(np.array(image), cls=True)
-        ocr_layout = parse_ocr_data_paddle(ocr_data)
-        text_from_ocr = ""
-        for text_region in ocr_layout:
-            text_from_ocr += text_region.text
+
+    if ocr_agent == OCR_AGENT_PADDLE:
+        ocr_regions = get_ocr_regions_paddle(image)
     else:
-        text_from_ocr = unstructured_pytesseract.image_to_string(
-            np.array(image),
+        ocr_regions = get_ocr_regions_tesseract(image, ocr_languages)
+
+    return ocr_regions
+
+
+def get_ocr_regions_tesseract(
+    image: PILImage,
+    ocr_languages: str = "eng",
+) -> List[TextRegion]:
+    """Get the OCR regions from image as a list of text regions with tesseract."""
+
+    logger.info("Processing OCR with tesseract...")
```

Comment:
```suggestion
    logger.info("Processing entire page OCR with tesseract...")
```
to be consistent with paddle